sonic-net/sonic-buildimage
# [sfp-refactoring] Initial support for CMIS application initialization

[sfp-refactoring] Quanta IX9: migrate to the new sfp-refactoring framework

Migrate the Quanta IX9 to the new sfp-refactoring framework

Signed-off-by: Dante Su <dante.su@broadcom.com>



#### Why I did it
Migrate the Quanta IX9 to the new sfp-refactoring framework

#### How I did it
Update the sfp.py of Quanta IX9 to use the new sfp-refactoring framework

#### How to verify it
1. Bring up Quanta IX9 with the default configuration and CMIS 4.0/5.0 QSFPDD modules
2. Issue 'sudo sfputil show eeprom --dom' to verify both the static EEPROM and DOM information
3. Issue 'sudo sfputil show error-status -hw' to verify error states

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog

<table>
<tr>
<th>Repo</th>
<th>PR title</th>
<th>State</th>
</tr>
<tr>
<td>SONiC</td>
<td><a href='https://github.com/Azure/SONiC/pull/876'>[sfp-refactoring] Initial support for CMIS application initialization</a></td>
<td><img src="https://img.shields.io/github/pulls/detail/state/Azure/SONiC/876"></td>
</tr>
<tr>
<td>sonic-buildimage</td>
<td><a href='https://github.com/Azure/sonic-buildimage/pull/9157'>[sfp-refactoring] Initial support for CMIS application initialization</a></td>
<td><img src="https://img.shields.io/github/pulls/detail/state/Azure/sonic-buildimage/9157"></td>
</tr>
<tr>
<td>sonic-platform-common</td>
<td><a href='https://github.com/Azure/sonic-platform-common/pull/219'>[sfp-refactoring] Initial support for CMIS application initialization</a></td>
<td><img src="https://img.shields.io/github/pulls/detail/state/Azure/sonic-platform-common/219"></td>
</tr>
<tr>
<td>sonic-platform-daemon</td>
<td><a href='https://github.com/Azure/sonic-platform-daemons/pull/217'>[sfp-refactoring] Initial support for CMIS application initialization</a></td>
<td><img src="https://img.shields.io/github/pulls/detail/state/Azure/sonic-platform-daemons/217"></td>
</tr>
<td>sonic-utilities</td>
<td><a href='https://github.com/Azure/sonic-utilities/pull/1855'>[sfp-refactoring] Initial support for CMIS application initialization</a></td>
<td><img src="https://img.shields.io/github/pulls/detail/state/Azure/sonic-utilities/1855"></td>
</table>

#### A picture of a cute animal (not mandatory but encouraged)
